### DISCUSSION TITLE
[Feature] Send to Archive based on Date

### DISCUSSION THREAD

**Discussion by thomaslc66** [immich-app/immich#8788] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

A cool feature, that would likely be close to this one: https://github.com/immich-app/immich/discussions/1673

The goal would be to select in the Archive a date and to send all pictures that are taken before this date to the archive.
Like, send to archive picture taken "before 01.01.2022". Like that, you have an auto archiver.

With the future ability to synch back and forth, it could help for example people to keep old pictures in the server without taking too much space on their phone and still can backup the old pictures.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
This'd be covered by smart rules, so I'm gonna close this thread in favour of that one.
